### DISCUSSION TITLE
List of supported batteries

### DISCUSSION THREAD

**Discussion by MUYUM** [fl4p/batmon-ha#253] [Q&A]
There is a lot of different batteries on the market. I'm changing a batterie in my RV and I've already spend quite some time seraching for a battery, which could be integrated with Batmon.

 I would like to have a list of batteries (battery type, manufacturer, image, BMS type, description...), so we could have an overview of batteries, which offer bluetooth compatibility to integrate it to Batmon.

For example:

**Supported:**
Manufacturer | Voltage | Capacity (Ah) | Type | Description
--- | --- | --- | --- |--- 
VATRER | 12.8 | 100Ah | AppConnection (BT) | Xiaoxiang BMS

**Not supported:**
Manufacturer | Voltage | Capacity (Ah) | Type | Description
--- | --- | --- | --- |--- 
LiTime | 12.8 | 100Ah | Smart (with BT) | Own BMS, which is not listed anywhere

I think that comunity could contribute quite some info and also manufacturers would be more keen to have batteries compatible with HA.

**Comment by garylankford8** [comment]


**Reply by Karatzie** [reply]
Hi. @garylankford8 .

type dummy does not communicate over Bluetooth.
This is a simulated internal BMS of batmon-ha.

BR
Andreas

**Comment by AORDU** [comment]
yo tengo dos Baterias de Redodo de 100Ah con  BLe y no consigo  comunicarlar con BATMON  con ningun type  jk, jbd, ant, daly, supervolt, sok, victron or dummy.,  Alguien ha conseguido comunicar HA con las baterias de Redodo?
Antonio Orduña

**Comment by garylankford8** [comment]
> yo tengo dos Baterias de Redodo de 100Ah con  BLe y no consigo  comunicarlar con BATMON  con ningun type  jk, jbd, ant, daly, supervolt, sok, victron or dummy.,  Alguien ha conseguido comunicar HA con las baterias de Redodo?
> Antonio Orduña

English translation:
I have two 100Ah Redodo batteries with BLe and I cannot communicate with BATMON with any type jk, jbd, ant, daly, supervolt, sok, victron or dummy. Has anyone managed to communicate HA with Redodo batteries?
